hack/build.sh

```diff
@@ -61,6 +61,9 @@ run() {
     # Format source code
     go_fmt
 
+    # Cleanup imports
+    go_imports
```

Comment:
Minor... same as comment to `go_fmt`. I'd move anything that changes the code before compile and test.

Reply:
I will change the order in another PR, just let those get merged first.